[2022-10-03 07:29:11,154] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-10-03T07:27:38.538436+00:00 [queued]>
[2022-10-03 07:29:11,177] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-10-03T07:27:38.538436+00:00 [queued]>
[2022-10-03 07:29:11,178] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-03 07:29:11,178] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-10-03 07:29:11,179] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-03 07:29:11,219] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-10-03 07:27:38.538436+00:00
[2022-10-03 07:29:11,229] {standard_task_runner.py:52} INFO - Started process 228 to run task
[2022-10-03 07:29:11,236] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'manual__2022-10-03T07:27:38.538436+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmptnwfobou', '--error-file', '/tmp/tmpscxqasi_']
[2022-10-03 07:29:11,237] {standard_task_runner.py:77} INFO - Job 306: Subtask local_to_gcs_task
[2022-10-03 07:29:11,384] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-10-03T07:27:38.538436+00:00 [running]> on host 9469dba0111d
[2022-10-03 07:29:11,550] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-10-03T07:27:38.538436+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-03T07:27:38.538436+00:00
[2022-10-03 07:29:18,238] {python.py:175} INFO - Done. Returned value was: None
[2022-10-03 07:29:18,336] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20221003T072738, start_date=20221003T072911, end_date=20221003T072918
[2022-10-03 07:29:18,451] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-03 07:29:18,616] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
